accounts-cluster-bench/src/main.rs

```diff
@@ -291,6 +553,22 @@ fn run_accounts_bench(
         None,
     );
 
+    let exit = Arc::new(AtomicBool::new(false));
+    let base_keypair_pubkey = base_keypair.pubkey();
+    let rpc_bench_threads: Vec<_> = if let Some(rpc_benches) = rpc_benches {
+        make_rpc_bench_threads(
```

Comment:
These threads will start racing with account creation right? So stats.errors
wont' be very accurate since it'll depend on scheduling etc. Maybe move
starting the threads right after account creation?